cookbooks/travis_build_environment/recipes/hhvm.rb

```diff
@@ -3,7 +3,7 @@
   key = "E7D1FA0C"
 else
   hhvm_uri = 'http://dl.hhvm.com/ubuntu'
```

Comment:
Fred Emmot (of Facebook and author of the linked blog post) mentioned that 

> dl.hhvm.com now also supports HTTPS as well; if you wish to switch, you might need to install the apt-transport-https package first though.

I would be in favor of making the switch, WDYT?